### ISSUE TITLE
opportunity to display help information cross platform

### ISSUE THREAD

**Issue by mgreenegit** [PowerShell/Phosphor#17]
Phosphor could also provide a nice option to render help information in an easy to read format in a consistent way across platforms.  Examples of content could include get-help, about_ topics, or even markdown from powershell-docs.

**Comment by daviwil** [comment]
Yep, it'd be good to provide a cross-platform replacement for `Get-Help -ShowWindow`!
